packages/entities/entities-shared/src/composables/useExternalLinkCreator.ts

```diff
@@ -0,0 +1,18 @@
+export default function useExternalLinkCreator(routeParams: string[]): string {
```

Comment:
needs a unit test

Reply:
Added some basic tests. Because we create the base url inside `useExternalLinkCreator`, the only way to make this function fail is by providing an empty route param, or one that doesn't match the regex I defined.


(Note: I used the same regex as https://github.com/Kong/public-ui-components/blob/main/packages/core/forms/src/generator/utils/validators.ts#L209)